scripts/ta_bin_to_c.py

```diff
@@ -37,6 +39,25 @@ def get_args():
 
     return parser.parse_args()
 
+def get_name(obj):
+    # Symbol or section .name might be a byte array or a string, we want a
+    # string
+    try:
+        name = obj.name.decode()
+    except (UnicodeDecodeError, AttributeError):
+        name = obj.name
+    return name
+
+def ta_get_flags(ta_f):
+
```

Comment:
empty line can be removed.

Reply:
True, move it up because one more empty line is needed before a function definition (`pycodestyle` rule, also at lines 42 and 62 -- see Travis CI).

Reply:
@etienne-lms @jforissier  here also I have question - did you changed code style for project? All other functions in this file have def name(); empty line; (function body). Why this one should be different?

Reply:
We've been running `pycodestyle` for almost 1.5 year (commit a3a309a86) and just before that I re-formatted `scripts/ta_bin_to_c.py` (commit c51c4e1da). This sets the rule for two blank lines before a function.

The blank line after the function name however is not a `pycodestyle` thing it seems, in other words you may or may not have one it doesn't matter. But, if you consider all the Python scripts in `scripts/` and not just `ta_bin_to_c.py`, a majority of functions don't start with a blank line. So it's better to unify towards this style.